src/neptune/cli/containers.py

```diff
@@ -131,14 +146,29 @@ def sync(self, *, backend: "NeptuneBackend", container_id: UniqueId, container_t
                     except NeptuneConnectionLostException as ex:
                         if time.monotonic() - start_time > retries_timeout:
                             raise ex
-                        logger.warning(
-                            "Experiencing connection interruptions."
-                            " Will try to reestablish communication with Neptune."
-                            " Internal exception was: %s",
-                            ex.cause.__class__.__name__,
-                        )
-
-    def move(self, *, base_path: Path, target_container_id: UniqueId, container_type: ContainerType) -> None:
+                        elif ex.cause.__class__.__name__ == "HTTPTooManyRequests":
+                            warn_once(
+                                "Looks like you're reaching the default workspace logging-rate limit."
```

Comment:
To simplify slightly: 
You're about to reach the default logging-rate limit for your workspace.
See how to optimize the logging calls to reduce requests: https://docs.neptune.ai/help/reducing_requests/
To increase the limit for your workspace, contact sales@neptune.ai.

Reply:
Thanks!
Instead of "You're about to reach ...", I rephrased it to "You're hitting the ..." as this error is shown only one the limit has been breached.